docs/recipes/WritingTests.md

```diff
@@ -49,125 +76,26 @@ Then, add this to `scripts` in your `package.json`:
 
 and run `npm test` to run it once, or `npm run test:watch` to test on every file change.
 
-### Action Creators
-
-In Redux, action creators are functions which return plain objects. When testing action creators, we want to test whether the correct action creator was called and also whether the right action was returned.
-
-#### Example
-
-```js
-export function addTodo(text) {
-  return {
-    type: 'ADD_TODO',
-    text
-  }
-}
-```
-
-can be tested like:
+### Action Creators & Async Action Creators
```

Comment:
:hand: I'd prefer _not_ to use the phrase "async action creators" - I've been trying to drop that terminology from the docs.  Let's call them "Thunks" instead

Reply:
Done, including an instance of "created async thunk action creator" I had added 😅